Tests/Indicators/SmoothedForceIndexTests.cs

```diff
@@ -0,0 +1,37 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using NUnit.Framework;
+using QuantConnect.Data.Market;
+using QuantConnect.Indicators;
+
+namespace QuantConnect.Tests.Indicators
+{
+    [TestFixture]
+    public class SmoothedForceIndexTests : CommonIndicatorTests<TradeBar>
```

Comment:
It seems one test is hanging and some others are failing 
![image](https://github.com/user-attachments/assets/ead59d4f-7b64-49f6-8045-fc7ab0e5c5bf)
![image](https://github.com/user-attachments/assets/e7071e39-1869-44a7-a857-45951926d99b)